draft-bormann-gendispatch-with-expert-review.md

```diff
@@ -139,6 +152,8 @@ the registration.
 This document updates {{RFC7120@BCP100}} to apply to the augmented policies
 defined above in {{rfcreq}}, {{ietfrev}}, and {{stdsact}}.
 
+(TODO: Explain how a document can specify this before BCP100 has been updated.)
+
```

Comment:
Just a proposal: for the period of time while BCP 100 is not updated in this respect, authors of specifications can use text that builds on [1], in a section that establishes a new registry using one of the augmented registration policies.

> The procedure for early IANA allocation of "standards track code points" defined in [RFC7120] also applies. When such a procedure is used, IANA will ask the designated expert(s) to approve the early allocation before registration. In addition, working group chairs are encouraged to consult the expert(s) early during the process outlined in Section 3.1 of [RFC7120].

[1] https://www.ietf.org/rfc/rfc9668.html#section-8.3